src/Bicep.Core/Workspaces/SourceFileGroupingBuilder.cs

```diff
@@ -3,43 +3,81 @@
 using System;
 using System.Collections.Generic;
 using System.Collections.Immutable;
-using System.Diagnostics.CodeAnalysis;
 using System.Linq;
 using Bicep.Core.Diagnostics;
 using Bicep.Core.FileSystem;
 using Bicep.Core.Parsing;
-using Bicep.Core.Syntax;
+using Bicep.Core.Registry;
 using Bicep.Core.Utils;
+using Bicep.Core.Workspaces;
 using static Bicep.Core.Diagnostics.DiagnosticBuilder;
 
-namespace Bicep.Core.Workspaces
+namespace Bicep.Core.Syntax
```

Comment:
namespace change intentional?

Reply:
Nope - not really sure how this happened. I'll revert it back...